server/memphis_handlers_tags.go

```diff
@@ -37,18 +37,12 @@ type TagsHandler struct{ S *Server }
 
 func CreateTag(name string, entity_type string, entity_id primitive.ObjectID, color string) error {
 	name = strings.ToLower(name)
-	exist, _, err := IsTagExist(name)
-	if err != nil {
-		return err
-	}
-	if exist {
-		return nil
-	}
+	entity := strings.ToLower(entity_type)
 	var newTag models.Tag
 	stationArr := []primitive.ObjectID{}
 	schemaArr := []primitive.ObjectID{}
 	userArr := []primitive.ObjectID{}
-	switch entity_type {
+	switch entity {
```

Comment:
you need to validate the entity type which will return 666 error indicates. that it is not a supported type